site.go

```diff
@@ -123,6 +124,14 @@ func (s *Site) IsValid() error {
 		}
 	}
 
+	if s.ResizingService != "imgix" && s.ResizingService != "thumbor" && s.ResizingService != "" {
+		return errors.New("Unrecognized/Unimplemented resizing service")
```

Comment:
If you could add the name of the unrecognized service in the error message, that would be great as it would make the error clearer. Plus maybe a list of recognized service names if you think that's helpful.